dags/process_elife_zip_dag.py

```diff
@@ -4,10 +4,12 @@
 """
 import logging
 import re
+from concurrent.futures.thread import ThreadPoolExecutor
```

Comment:
I've used this to [parallelize loading HTTP images and similar in end2end testing](https://github.com/elifesciences/elife-spectrum/blob/54626dae48673056febdced18ca9505f44c5a30a/spectrum/checks.py#L907)